scripts/build-images.sh

```diff
@@ -22,15 +22,16 @@ build () {
 		--build-arg=REFS="$REF" \
 		--build-arg=BRANCH="$BRANCH" \
 		--build-arg OPENSCAD_VERSION="$VERSION" \
+		--build-arg BUILD_TYPE="Release" \
 		$DOCKERFILE_ARGS \
 		"$DIR"
 }
 
 V=$(git log -1 --date="format:%Y.%m.%d.dd%j%H" --format="%cd")
 
-#build	tags	openscad-2015.03	2015.03	""	openscad/buster
-build	tags	openscad-2019.05	2019.05	""	openscad/buster
-build	tags	openscad-2021.01	2021.01	""	openscad/buster
-build	heads	master			dev	"$V"	openscad/bookworm
+#build	tags	openscad-2015.03	2015.03	""	openscad/buster Dockerfile
+#build	tags	openscad-2019.05	2019.05	""	openscad/buster Dockerfile
```

Comment:
This shouldn't have been added. Undo this line

Reply:
Thanks, we can change the scripts, e.g. adding parameters that are translated into `docker build` arguments, but the default should stay configured for running the official builds. Or, if that helps, we can have a separate "build all" script. Basically I just need a place and setup to make my life easier maintaining the builds, otherwise that's not feasible as one person effort.